### PR TITLE
Styles nav pills stacked menu

### PR DESCRIPTION
**Pull request by slobdog**

### CODE REVIEW

File: less/navs.less [hunk 1]
```diff
@@ -149,10 +149,41 @@
 .nav-stacked {
   > li {
     float: none;
+
+    > a {
+      font-size: @font-size-base;
```
Comment (StevePole) [inline less/navs.less:154]: Any reason for this font-size rule, won't it be this size anyway?

File: less/transferwise.less [hunk 2]
```diff
@@ -56,7 +56,7 @@ a.btn {
 }
 
 .nav > li > a {
-    padding: 11px 16px 9px;
+    padding: 11px 16px 5px;
```
Comment (StevePole) [inline less/transferwise.less:59]: Why was this one necessary?  It could have knock on effects in other areas.

File: less/navs.less (hunk 1 above)
Comment (slobdog) [inline less/navs.less:154]: for some reasons we use 14px font-size in our bootstrap theme
you can have a look at nav.less line 24 "font-size: @font-size-small;"

File: less/transferwise.less (hunk 2 above)
Comment (slobdog) [inline less/transferwise.less:59]: Very important rule is to have hight exactly 40px but we have 44px all over the project. And i've changed bottom padding to make icon and text look more in centre

Comment (StevePole) [inline less/transferwise.less:59]: ok, I checked this.  I've actually removed this rule from that file in my latest set of local changes, however, there is a corresponding rule in navs.less, with padding: 10px 16px;  We need to keep the spacing even either side, I'd suggest changing the version in navs.less to 8px 16px and making a rule to reposition the glyphicon in that context e.g ... > li > a > .glyphicon { ... }  Ok to make that change?

**Comment by StevePole** [comment]
:+1:

**Pull request merged**